.github/workflows/ci.yml

```diff
@@ -23,19 +23,25 @@ jobs:
           key: yarn-${{ hashFiles('yarn.lock') }}
           restore-keys: |
             yarn-
+      - name: 'Check or update Yarn cache (fix w/ "yarn install")'
+        env:
+          YARN_ENABLE_SCRIPTS: false # disable post-install scripts
+          YARN_NODE_LINKER: pnp # use pnp linker for better linking performance: it's meant to update yarn cache only
+        run: |
+          yarn install --immutable --skip-builds
```

Comment:
This cannot be moved to the other job, since the yarn cache is cached for the build/test jobs.